uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -187,11 +187,11 @@ sub IsCandidateInfoValid {
         #1) Check to see if the file is of type DICOM###########
         #2) Check to see if the header matches the patient-name#
         ########################################################
-        if ( ( $_ ne '.' ) && ( $_ ne '..' ) ) {
+        if ( ( $_ ne '.' ) && ( $_ ne '..' ) && (basename($_) ne '.DS_Store')) {
```

Comment:
Yes, the one dot and two dot are file system convention refer to the current directory and the parent directory respectively.

that's why when you do cd ..  
it brings you to the parent directory.